integration/test/ParseUserTest.js

```diff
@@ -979,11 +979,11 @@ describe('Parse User', () => {
     await Parse.FacebookUtils.link(user);
 
     expect(Parse.FacebookUtils.isLinked(user)).toBe(true);
-    expect(Parse.AnonymousUtils.isLinked(user)).toBe(true);
+    expect(Parse.AnonymousUtils.isLinked(user)).toBe(false);
```

Comment:
It's interesting though that there is even a test case that expects anonymous user to be `true` after `linkWith`. Did you take a look at the commit history / PR that added the test, so see if that was deliberately, or just a mistake?

Reply:
Never mind, I just noticed that you were also the author if this test in https://github.com/parse-community/Parse-SDK-JS/commit/ba6c448d4c79ceff7ceda54edcf1712d90e2d5d0. There is no further history to it.